utils/parseAttributes.ts

```diff
@@ -11,16 +11,16 @@ export function parseAttributes<
   };
 }
 
-function removeEmptyValues(data: NFTMetadataInput["attributes"]) {
+export function removeEmptyValues(data: NFTMetadataInput["attributes"]) {
   if (!data) {
     return data;
   }
   if (Array.isArray(data)) {
     const parsedArray = data
       .filter(({ value }) => value !== "")
       .map(({ value, trait_type }) => ({
-        value,
         ...(!!trait_type && { trait_type }),
+        value,
```

Comment:
why change the order here?

Reply:
makes more sense in the preview of the batch upload seeing trait_type first

Reply:
ah so just visual ordering, ok